src/Symfony/Component/Form/Tests/Extension/Core/Type/DateTimeTypeTest.php

```diff
@@ -263,6 +263,7 @@ public function testSubmitDifferentPattern()
     }
 
     // Bug fix
+
```

Comment:
Looks wrong to me. I would instead remove the comment above which does not really add any value.